server/src/main/java/org/opensearch/search/pipeline/Pipeline.java

```diff
@@ -117,7 +118,7 @@ protected void afterResponseProcessor(Processor processor, long timeInNanos) {}
 
     protected void onResponseProcessorFailed(Processor processor) {}
 
-    SearchRequest transformRequest(SearchRequest request) throws SearchPipelineProcessingException {
+    SearchRequest transformRequest(SearchRequest request, Map<String, Object> requestContext) throws SearchPipelineProcessingException {
```

Comment:
should we pass this context to SearchPhaseResults Processor too? I think we should. Whats your thought? @msfroh

Reply:
Yes. I can make that change too.